src/makeRobotAccountant.js

```diff
@@ -6,7 +6,25 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let callCount = 0;
+
+  const f = (x) => {
+    const g = (y) => {
+      callCount++;
+
+      if (callCount % 4 === 0) {
```

Comment:
I'm not sure about this condition, looks like it wont works correct for  all numbers. I suggest to check is callCount > 3 and only then check if callCount % 2 === 0 and only in this case show the error